docs/contributing/Tree-hygiene.md

```diff
@@ -84,29 +84,58 @@ _See also: [What should I work on?](What-should-I-work-on.md)_
-* only affect code inside the `.github` directory, `.cirrus.yml` or `.ci.yaml` config files.
+* only affect code inside the `.github` directory or `.ci.yaml` config files.
 * only affect `.md` files.
 * are generated by automated bots (rollers).
-* have an _explicit_ exemption (a message from a member of `@test-exemption-reviewer` on Discord starting with `test-exempt:`).
-<!-- update https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/service/config.dart if you change who is on the list -->
 
-**In the repositories listed above, a bot will comment on your PR if you need an explicit exemption.** Do not land a PR that has had such a message from the bot unless it has an explicit exemption! The message tells you how to ask for an exemption. Please follow those instructions.
+If a reviewer says a PR should have a test, then it needs a test regardless of the exemptions above.
+
+Feel free to update the bot's logic to catch more cases that should be
+automatically exempt ([in the cocoon repo](https://github.com/reidbaker/cocoon/blob/main/app_dart/lib/src/request_handlers/github/webhook_subscription.dart)).
 
-**If the PR is in a repository that is not listed above, meaning is not supported by the bot, then it is the responsibility of the pull request reviewer to make sure that tests have been added to support the code change.**
+> [!NOTE]
+> PRs adding data-driven fixes require tests that fall under the test_fixes
+> directory, but are _not_ yet recognized by the bot as being tested.
 
-In particular, the following kinds of PRs are *not* automatically exempt and require an explicit comment even though the answer may be obvious: refactors with no semantic change (e.g. null safety migrations), configuration changes in the aforementioned repos, changes to analysis (fixing lints, turning on lints), changes to test infrastructure, manual dependency rolls, fixes to existing tests, cosmetic fixes to unpublished example apps.
+The following kinds of PRs are *not* automatically exempt and require an exemption even if the answer may be obvious:
 
-If a reviewer says a PR should have a test, then it needs a test regardless of the exemptions above.
+* refactors with no semantic change (e.g. null safety migrations)
+* configuration changes in the aforementioned repos
+* changes to analysis (fixing lints, turning on lints)
+* changes to test infrastructure
+* manual dependency rolls
+* fixes to existing tests
+* cosmetic fixes to unpublished example apps.
+
+**In the repositories listed above, a bot will comment on your PR if you need an explicit exemption.**
+
+> [!IMPORTANT]
+> If the PR is in a repository that is not listed above, meaning is not
+> supported by the bot, then it is _still_ a responsibility of the pull request
+> reviewer to make sure that tests have been added to support the code change.
+
+### Test Exemptions
+
+A `@test-exemption-reviewer` member on Discord can exempt a PR by commenting
```

Comment:
Can `Discord` link to the contributing/chat.md page?

Reply:
Done!